python/ray/serve/BUILD

```diff
@@ -175,6 +175,21 @@ py_test(
     deps = [":serve_lib"],
 )
 
+py_test(
+    name = "test_pydantic_serialization",
+    size = "small",
+    srcs = serve_tests_srcs,
+    tags = ["exclusive"],
+    deps = [":serve_lib"],
+)
```

Comment:
In the future it'd be better to make a separate PR for this. Makes it easier to review and merged with fewer moving pieces.